commands/highlight.js

```diff
@@ -58,7 +59,7 @@ const addHighlight = async (keywords, user) => {
 		}
 	}).then((count) => {
 		if (count != 0) {
-			user.send('Attempted to add **`' + keywords + '`** to your highlights, but it\'s already there!');
+			discordDMWrapper.sendMessage(user, `Attempted to add '${keywords}' to your highlights, but it's already there!`).catch(() => {});
```

Comment:
Can we get error messages here as well? Just a short one like above, just wherever they send the message "Message failed, DMs closed" essentially. Rather than having empty catch functions.

Reply:
Fixed. See new code.